app/helpers/bootstrap_form_builder.rb

```diff
@@ -0,0 +1,25 @@
+class BootstrapFormBuilder < ActionView::Helpers::FormBuilder
+  def text_field(attribute, options = {})
+    options[:class] = options[:class].present? ? "#{options[:class]} form-control" : 'form-control'
+    unless @object[attribute].nil?
+      options[:class] += (@object.errors[attribute].any? ? ' is-invalid' : ' is-valid')
+    end
+    super(attribute, options)
```

Comment:
Skaties cienajamais, tev vajadzetu izdarit jaunu input fieldu, pieņemsim: text_input un taja ielīkt:
1. label ar bootstrap tipu
2. šo text_field logiku
3. Ielikt logiku kad jāatspoguļot errorus zem laukā

Pēc tam tu varesi to turpmak visur izmantot un ekonomet vietu